HipChatPS/Public/Send-HipChat.ps1

```diff
@@ -25,14 +30,22 @@
         #Set whether or not this message should trigger a notification for people in the room. (default: false)
         [switch]$notify,
 
+        #Required. Server name, default to 'api.hipchat.com'
+        [Parameter(Mandatory = $True)]
```

Comment:
Can't be mandatory = true and have a default. Suggest you remove the 'mandatory = true'.

Reply:
I fully agree.
This would normally be tested in the CI; but this is not yet implemented in the project.

I created #12 to fix this.

Reply:
Actually there is CI already on this project, it runs from my AppVeyor account still. The tests are currently failing because `-server` is mandatory and not explicitly declared in the tests.

Reply:
ok... I meant:

The CI and the tests are not yet aligned with the rest of the projects and the branches to not forbid merging while the CIs do not pass.

Reply:
fixed it

Reply:
Ah I see, thanks for the clarification :)